app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def recent_posts
+    Post.where(author_id: id).limit(3).order(created_at: :desc)
+  end
```

Comment:
- [ ]  Another way of doing this would be:
```rb
def recent_post
  posts.order('created_at Desc').limit(3)
end
```
Remember that the posts have been already associated with the users.

Reply:
✅ Resolved.